packages/python-packages/apiview-copilot/evals/run.py

```diff
@@ -19,15 +18,19 @@
         "-n",
         type=int,
         default=1,
-        help="The number of runs to perform, with the median of results kept. Defaults to 3.",
+        help="The number of runs to perform, with the median of results kept. Defaults to 1.",
     )
     parser.add_argument(
         "--test-file",
         "-t",
         type=str,
-        default="reviews.jsonl",
-        help="Only run a particular jsonl test file, takes the name or path to the file. Defaults to all.",
+        required=True,
+        help="Path to workflow YAML or JSONL file name.",
```

Comment:
How does targeting the JSONL work? Does it look for a matching YAML file, or something else?